packages/plugin-commands-script-runners/src/exec.ts

```diff
@@ -54,11 +58,19 @@ For options that may be used with `-r`, see "pnpm help recursive"',
             name: '--recursive',
             shortAlias: '-r',
           },
+          {
+            description: 'If exist, runs file inside of a shell. \
+Uses /bin/sh on UNIX and \\cmd.exe on Windows. \
+A different shell can be specified as a string. \
```

Comment:
you're saying that it can be a string as well but at line 35 I see that `shell-mode` is boolean, so it cannot be a string.

I don't think a CLI option can be both a string and a boolean. If it can be a string then `pnpm -r --shell-mode exec` will be parsed as if exec is the value of shell-mode.

Reply:
Sorry, I didn't read it carefully. I copied the quote directly from https://github.com/sindresorhus/execa/tree/e4b929547ade335b0dd9d7afe1a35989e19fbc8a#shell

I will change it now